### PR TITLE
doc: Add new warning when to use KVM

### PR DESCRIPTION
**Pull request by rafarosa92**
When using KVM, you may encounter issues even if your system appears to be okay. In such cases, using sudo can often resolve the problem.

### CODE REVIEW

**Comment by rafarosa92** [comment]
Yes, I checked, my user and the docker group are part of sudo, I know it's not a good practice to use sudo but this was the way it worked, I can perform other tests here but to simulate the error and even debug the image better...

**Comment by rafarosa92** [comment]
Same 660

**Comment by rafarosa92** [comment]
Yes, it works. Do you want decline the pull request, or would you like me to change it and add this warning?

File: readme.md [hunk 1]
```diff
@@ -49,19 +49,19 @@ docker run -it --rm -p 8006:8006 --device=/dev/kvm --cap-add NET_ADMIN --stop-ti
 
 ## FAQ
 
-* ### How do I use it?
+- ### How do I use it?
 
   Very simple! These are the steps:
-  
+
   - Start the container and connect to [port 8006](http://localhost:8006) using your web browser.
```
Comment (Sonu7218) [inline readme.md:56]: Hii

**Review by Sonu7218 — COMMENTED** [review]
* ### How do I select the Windows version?

File: readme.md (hunk 1 above)
Comment (FikriRadsoft) [reply readme.md:56]: add the --env=VERSION=x
which x is the windows version, and the list is available in the readme

**Pull request closed without merging**